src/test/net_tests.cpp

```diff
@@ -318,28 +318,30 @@ BOOST_AUTO_TEST_CASE(cnetaddr_tostring_canonical_ipv6)
         {"2001:db8:aaaa:bbbb:cccc:dddd:eeee:AaAa", "2001:db8:aaaa:bbbb:cccc:dddd:eeee:aaaa"},
     };
     for (const auto& [input_address, expected_canonical_representation_output] : canonical_representations_ipv6) {
-        CNetAddr net_addr;
-        BOOST_REQUIRE(LookupHost(input_address, net_addr, false));
-        BOOST_REQUIRE(net_addr.IsIPv6());
-        BOOST_CHECK_EQUAL(net_addr.ToString(), expected_canonical_representation_output);
+        const std::optional<CNetAddr> net_addr{LookupHost(input_address, false)};
+        BOOST_REQUIRE(net_addr.value().IsIPv6());
+        BOOST_CHECK_EQUAL(net_addr.value().ToString(), expected_canonical_representation_output);
     }
 }
-
+/*
```

Comment:
I think this comment was not meant to be committed?

Reply:
Sorry, I remove this PR from draft before pushing more changes.. my fault!